packages/schemas/src/foundations/jsonb-types.ts

```diff
@@ -29,8 +29,16 @@ export const oidcModelInstancePayloadGuard = z
 
 export type OidcModelInstancePayload = z.infer<typeof oidcModelInstancePayloadGuard>;
 
+// Import from @logto/core-kit later, pending for new version publish
+export const redirectUriRegEx = /^https?:\/\/[\d&./:=?_a-z-]+$/;
+export const nativeRedirectUriRegEx = /^[a-z][\d_a-z]*(\.[\d_a-z]+)+:\/\/[\d/_a-z-]+$/;
```

Comment:
should allow uppercase

Reply:
Revised and using URL constructor instead. Please take a look